### ISSUE TITLE
Unknown provider: ui.dateProvider when using AngularJS 1.2.1

### ISSUE THREAD

**Issue by Splaktar** [angular-ui/ui-date#52]
Error: [$injector:unpr] Unknown provider: ui.dateProvider <- ui.date
http://errors.angularjs.org/1.2.1/$injector/unpr?p0=ui.dateProvider%20%3C-%20ui.date
    at http://localhost:8888/lib/angular.js:78:12
    at http://localhost:8888/lib/angular.js:3451:19
    at Object.getService [as get](http://localhost:8888/lib/angular.js:3578:39)
    at http://localhost:8888/lib/angular.js:3456:45
    at getService (http://localhost:8888/lib/angular.js:3578:39)
    at invoke (http://localhost:8888/lib/angular.js:3600:13)
    at Object.instantiate (http://localhost:8888/lib/angular.js:3636:23)
    at http://localhost:8888/lib/angular.js:6639:28
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.2.1/angular-route.min.js:6:476
    at publicLinkFn (http://localhost:8888/lib/angular.js:5443:29) 

Currently using:
    <link rel="stylesheet" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/themes/smoothness/jquery-ui.min.css">

<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.1/angular.min.js"></script>

<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.1/angular-route.min.js"></script>

<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.1/angular-resource.min.js"></script>

<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.1/angular-sanitize.min.js"></script>

<script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>

<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>

app.js:
var app = angular.module('app',[
    'ngRoute',
    'ngResource',
    'Controllers',
    'ui.date'
]);

controller.js:
Controllers.controller('MapController', ['$scope', '$http', 'ui.date', function($scope, $http) {
...
}]);

index.html:
    <label>Start Date: <input type="text" ui-date ng-model="startDate"></label>
    <label>End Date: <input type="text" ui-date ng-model="endDate"></label>

The labels and input fields load with the page but date picking doesn't work and it stops the rest of the page from loading.
